de.dlr.sc.virsat.model.extension.ps.test/src/de/dlr/sc/virsat/model/extension/ps/util/test/ProductStructureHelperTest.java

```diff
@@ -37,14 +33,12 @@
 
 /**
  * Test Case for Generating Tree
- * @author bell_er
  *
  */
 public class ProductStructureHelperTest extends AConceptProjectTestCase {
-	private static final String CONCEPT_ID_EGSCC = de.dlr.sc.virsat.model.extension.ps.Activator.getPluginId();
-	private static final String EXTENSION_ID = "de.dlr.sc.virsat.model.Concept";
-
-	private static final int THREE = 3;
+	private static final String CONCEPT_ID_PS = Activator.getPluginId();
+	
+	private static final int AMOUNT_CT_SUBSYTEMS = 3;
 
 	ElementDefinition ed;

```

Comment:
I think we can make these private.

Reply:
arent they automatically private if not explicitly stated?

Reply:
The visibility level if not stated is considered "package protected". E.g. classes in the same package can access the variables. https://docs.oracle.com/javase/tutorial/java/javaOO/accesscontrol.html 

This has the minor consequence that eclipse doesnt create warnings for unused variables. So if, for example due to a change in the test case, the variable ed is no longer used, then eclipse would not create a warning. And the dead variable would just lie around.

Reply:
ok, than i add it to the change.